test/Spago/Publish.purs

```diff
@@ -85,133 +89,100 @@ spec = Spec.around withTempDir do
       spago [ "fetch" ] >>= shouldBeSuccess
       spago [ "publish", "-p", "root", "--offline" ] >>= shouldBeFailureErr (fixture "publish/1307-publish-dependencies/expected-stderr.txt")
 
-  Spec.describe "transfer" do
```

Comment:
GitHub doesn't show this diff in its most sensical way. The `transfer` section has been moved out to `Transfer.purs` and two tests right after it were indented one level to bring them back under `publish`.

This diff is best viewed with whitespace ignored.